72pe.typ

```diff
@@ -168,9 +261,130 @@
 // -----------------------------------------------------------------------------
 // Ground Checklists and Information page
 // -----------------------------------------------------------------------------
-#let ground_checklists_and_info = [
+#let ground_checklists_and_info_columns = [
+	#checklist("Preflight", palette.brown,
+		checklist_group("Fluids"),
+		([Engine master], [OFF]),
+		([Fuel quantity], [CHECK VISUALLY]),
+		([Fuel sumps], [DRAIN],
+		 [Left wing, right wing, fuel strainer. Inspect for contamination.]),
```

Comment:
Should mention the number of sumps in each area. IIRC it's 5 on each wing + 2 under the nose.